playbook.yaml

```diff
@@ -0,0 +1,72 @@
+---
+- name: Configure TODO App
+  hosts: webservers
+  remote_user: ec2-user
+
+  vars_prompt:
+    - name: API_KEY
+      prompt: Trello API Key?
+      private: true
+
+    - name: API_TOKEN
+      prompt: Trello API Token?
+      private: true
+
+    - name: BOARD_ID
+      prompt: Trello Board ID?
+      private: true
+
+    - name: LIST_ID
+      prompt: Trello List ID?
+      private: true
+
+  tasks:
+    - name: Install Dependencies
+      become: true
+      yum:
+        name:
+          - git
+          - python3
+        state: present
+
+    - name: Install Poetry
+      shell: curl -sSL https://install.python-poetry.org | python3 -
+      args:
+        creates: ~/.local/bin/poetry
+
+    - name: Create TODO App Directory
+      become: true
+      file:
+        path: /opt/todoapp
+        state: directory
+        owner: ec2-user
+
+    - name: Clone TODO App
```

Comment:
This stuck out so hard to me like a leftover `#TODO` comment - brain couldn't parse it at first!